packages/tradershub/src/modals/AddOrManageAccount/CurrenciesForm.tsx

```diff
@@ -25,19 +31,34 @@ const CurrenciesForm = ({
     submitButtonLabel,
     isSubmitButtonDisabled = false,
     allCryptoCurrenciesAreAdded = false,
+    currentAccountCurrency,
+    addedFiatCurrency,
 }: TCurrenciesForm) => {
     const { isDesktop } = useDevice();
+    const { switchAccount, data } = useAuthorize();
+    const { data: activeTradingAccountData } = useActiveTradingAccount();
```

Comment:
```suggestion
    const { data: activeDerivTradingAccount } = useActiveTradingAccount();
```

Reply:
I think we can rename the hook itself with `useActiveDerivTradingAccount` it'll be easier for the consumers as well